src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -5,12 +5,16 @@ public class AuthenticationService {
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != null) {
+            return UserService.findByEmail(email).getPassword().equals(password);
+        }
```

Comment:
in this case, you call the findByEmail method twice with the same parameters. I think it is better to create a local change, because I can inject it into performance